indico/modules/events/registration/templates/display/participant_list.html

```diff
@@ -43,6 +46,14 @@
                         {% if show_affiliation %}
                             <td class="i-table">{{ personal_data.affiliation or '' }}</td>
                         {% endif %}
+                        {% if show_checkin_status %}
+                            <td class="i-table">
+                                {% if registration.checked_in
+                                      and registration.registration_form.publish_checkin_enabled %}
```

Comment:
no need to wrap it here - it probably looks better without and we have no hard length limit for jinja templates